### PR TITLE
Webex test-module bug

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37120

## Description
test module returns error - list index out of range

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
https://app.circleci.com/pipelines/github/demisto/content/86885/workflows/37cea260-b46f-41ff-9463-13f2f947aad3/jobs/359078 - test failed

File: Packs/CiscoWebExFeed/Integrations/CiscoWebExFeed/CiscoWebExFeed_test.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+DOMAIN_TABLE = [['Client Type', 'Domain(s)'],
+                ['domain1', '*.d1.com'],
+                ['domain2', '*.d2.com'],
+                ['Long message without domain name']]
+
+
+def test_grab_domains():
+    """
+    Given:
+        - Raw list of tuples that contains domain name and domain url, returned by api call
```
Comment (ShahafBenYakir) [inline Packs/CiscoWebExFeed/Integrations/CiscoWebExFeed/CiscoWebExFeed_test.py:10]: Can you elaborate on the different types of lines?

**Review by ShahafBenYakir — APPROVED** [review]
See my comment on the unit test file.
Plus, in order to be certain the change we are introducing in the docker image doesn't break anything, let's configure this feed and run it to be sure it works smoothly with the new docker image.

**Comment by daryakoval** [comment]
image
works great even with new docker

File: Packs/CiscoWebExFeed/ReleaseNotes/1_1_2.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cisco WebEx Feed
+- Fixed bug in *test-module*.
```
Comment (ShahafBenYakir) [inline Packs/CiscoWebExFeed/ReleaseNotes/1_1_2.md:4]:
```suggestion
- Fixed an issue where the integration failed to parse the domain list received.
```

**Pull request merged**